bitnami/spring-cloud-dataflow/templates/_helpers.tpl

```diff
@@ -276,7 +276,7 @@ Return the RabbitMQ Port
 */}}
 {{- define "scdf.rabbitmq.port" -}}
 {{- if .Values.rabbitmq.enabled }}
-    {{- printf "%d" (.Values.rabbitmq.service.port | int ) -}}
+    {{- printf "%d" (.Values.rabbitmq.service.port.amqp | int ) -}}
```

Comment:
I've just checked our RabbitMQ chart in order to verify this path and it seems it is **ports** (plural):

https://github.com/bitnami/charts/blob/ecdec9d9653fc8037762d2edcb777d885c2d8372/bitnami/rabbitmq/values.yaml#L851-L852

Hence, the correct change would be:
```suggestion
    {{- printf "%d" (.Values.rabbitmq.service.ports.amqp | int ) -}}
```